`set-contains` failing typecheck

The following program fails typechecking:
```
(sort ISet (Set i64))
(datatype SomeISet (Some ISet))
(rule ((Some seta) (set-contains seta x)) ())
```
with the message
```
[ERROR] In 3:1-45 of tempo.egg: (rule ((Some seta) (set-contains seta x)) ())
    Variable x is ungrounded
```

--- a/src/typechecking.rs
+++ b/src/typechecking.rs
@@ -842,7 +842,7 @@ pub enum TypeError {
     },
     #[error("{1}\nUnbound symbol {0}")]
     Unbound(String, Span),
-    #[error("{1}\nVariable {0} is ungrounded")]
+    #[error("{1}\nVariable {0} is ungrounded. A variable is grounded when it appears as an argument to a constructor or function in the query, not just under primitives or equalities.")]
     Ungrounded(String, Span),
     #[error("{1}\nUndefined sort {0}")]
     UndefinedSort(String, Span),

--- a/src/typechecking.rs
+++ b/src/typechecking.rs
@@ -842,7 +842,9 @@ pub enum TypeError {
     },
     #[error("{1}\nUnbound symbol {0}")]
     Unbound(String, Span),
-    #[error("{1}\nVariable {0} is ungrounded. A variable is grounded when it appears as an argument to a constructor or function in the query, not just under primitives or equalities.")]
+    #[error(
+        "{1}\nVariable {0} is ungrounded. A variable is grounded when it appears as an argument to a constructor or function in the query, not just under primitives or equalities."
+    )]
     Ungrounded(String, Span),
     #[error("{1}\nUndefined sort {0}")]
     UndefinedSort(String, Span),

